src/main/java/me/xmrvizzy/skyblocker/skyblock/item/exotic/DownloadColors.java

```diff
@@ -0,0 +1,25 @@
+package me.xmrvizzy.skyblocker.skyblock.item.exotic;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import me.xmrvizzy.skyblocker.utils.Http;
+
+import java.io.IOException;
+
+public class DownloadColors {
+    public static JsonObject ColorApiData;
+    public static void init() throws IOException {
+        try {
+            String url = "https://hysky.de/api/color"; // Item Api Request Url
+            String jsonResponse = Http.sendGetRequest(url);
+
+            ColorApiData = JsonParser.parseString(jsonResponse).getAsJsonObject();
+
+        } catch (IOException e) {
+            e.printStackTrace();
+        } catch (InterruptedException e) {
+            throw new RuntimeException(e);
+        }
+
+    }
+}
```

Comment:
This is not necessary, you can add hysky.de/api/color here: https://github.com/SkyblockerMod/Skyblocker/blob/d30994ef9a3208980f307d69db5e5cf0d42d10ef/src/main/java/me/xmrvizzy/skyblocker/skyblock/item/PriceInfoTooltip.java#L440

and also consider using https://github.com/AzureAaron/Skyblocker/blob/d30994ef9a3208980f307d69db5e5cf0d42d10ef/src/main/java/me/xmrvizzy/skyblocker/skyblock/item/PriceInfoTooltip.java#L411.